src/engine/benchmark/base/parseEvent_bench.cpp

```diff
@@ -97,9 +96,10 @@ static void copyEvents_batch(benchmark::State& state)
     std::transform(sampleEventsStr.begin(),
                    sampleEventsStr.end(),
                    std::back_inserter(events),
-                   [](const auto& e) { return base::parseEvent::parseOssecEvent(e); });
+                   [](const auto& e) { return base::parseEvent::parseWazuhEvent(e); });
 
-    for (auto _ : state) {
+    for (auto _ : state)
```

Comment:
Is it necessary to loop through state to get the number of elements?

Reply:
This seems to be only a coding style fix, the implementation is not part of this PR